src/components/Image/index.tsx

```diff
@@ -79,12 +70,7 @@ const Image: React.FC<ImageProps> = ({
   breakpoints,
   lightBoxWidth,
   lightBoxHeight,
-  lazyOptions = {
-    offsetBottom: 0,
-    offsetLeft: 0,
-    offsetRight: 0,
-    offsetTop: 0,
-  },
+  arcStatic = true,
```

Comment:
if this is not defined or null, then need to opt in to using arcStatic. need to test this further